core/src/main/java/org/apache/accumulo/core/spi/compaction/SimpleCompactionDispatcher.java

```diff
@@ -26,13 +28,14 @@
 
 /**
  * Dispatcher that supports simple configuration for making tables use compaction services. By
- * default it dispatches to a compaction service named default.
+ * default, it dispatches to the compaction service named default
+ * {@link org.apache.accumulo.core.Constants#DEFAULT_COMPACTION_SERVICE_NAME}.
```

Comment:
Could use the value javadoc tag here

```suggestion
 * {@value org.apache.accumulo.core.Constants#DEFAULT_COMPACTION_SERVICE_NAME}.
```